src/Evolution/DiscontinuousGalerkin/Actions/ComputeTimeDerivative.hpp

```diff
@@ -627,6 +634,125 @@ void ComputeTimeDerivative<Metavariables>::send_data_for_fluxes(
                            data));
       }
     }
+
+    if constexpr (Metavariables::local_time_stepping) {
+      using variables_tag = typename Metavariables::system::variables_tag;
+      using dt_variables_tag = db::add_tag_prefix<::Tags::dt, variables_tag>;
+      // We assume isotropic quadrature, i.e. the quadrature is the same in
```

Comment:
Is this an assumption, or is this checked anywhere? In other words, if I pass in a mixed quadrature, will there be an ASSERT elsewhere telling me to try again, or will the code proceed incorrectly?

Reply:
I'm not sure, probably depends on which different parts of the code are being used, but _a lot_ will break if you try isotropic quadrature. I can add an explicit check here if you want

Reply:
A check somewhere would be good I'd think. I'd suggest placing it the highest scope where it is still a concern.. though I'm not sure where that would be.

But this is just a suggestion, not a firm change request.